CHANGELOG.md

```diff
@@ -44,6 +44,17 @@ so it can't be used to retry failed downloads in `zebrad` 1.3.0 and earlier.
 
 We recommend upgrading to the latest Zebra release to avoid download issues in new installs.
 
+### Documentation Website Change
+
+We have replaced the API documentation on the [doc.zebra.zfnd.org](https://doc.zebra.zfnd.org)
+website with [docs.rs](https://docs.rs/releases/search?query=zebra). All links have been updated.
+
+Zebra's API documentation can be found on:
+- [`docs.rs`](https://docs.rs/releases/search?query=zebra), which renders documentation for the
```

Comment:
That sounds like a significant change, and I'm happy to do something like it in another ticket and PR.

What about linking to all the Zebra crates from [lib.rs](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/lib.rs), which appears as the first page at docs.rs/zebrad ?

Reply:
> I'm happy to do something like it in another ticket and PR.

> What about linking to all the Zebra crates from [lib.rs](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/lib.rs)

That sounds good, ideally we could nest the pub mod items in lib.rs under the application module too.

Reply:
I opened #7890 for linking to the Zebra crates from the `lib.rs` docs.

I'm not sure about the `application` module, I think the `config` could stay at the top level because it configures all of Zebra's crates (not just the configs in `zebrad`.)

I would also like to:
- delete the `prelude` module, it's not used much
- move `sentry` into components
- move `inbound`, `mempool`, and `sync` into their own crate, because they aren't application-specific, any Zebra binary or test might want to use them